src/main/java/software/amazon/cloudformation/resource/ResourceTypeSchema.java

```diff
@@ -153,6 +148,11 @@ public ResourceTypeSchema(Schema schema) {
             });
             return null;
         });
+
+        this.taggable = this.unprocessedProperties.containsKey("taggable")
```

Comment:
This may only be useful if the client will use `ResourceTypeSchema` and reply on this package.